controller/token.go

```diff
@@ -656,8 +631,8 @@ func modelToAppExternalToken(externalToken provider.ExternalToken, providerAPIUR
 	}
 }
 
-// GenerateUserToken obtains the access token from Keycloak for the user
-func GenerateUserToken(ctx context.Context, tokenEndpoint string, configuration LoginConfiguration, username string, userSecret string) (*app.AuthToken, error) {
+// ObtainKeycloakUserToken obtains the access token from Keycloak for the user
+func ObtainKeycloakUserToken(ctx context.Context, tokenEndpoint string, configuration LoginConfiguration, username string, userSecret string) (*app.AuthToken, error) {
```

Comment:
Why do we still need to obtain tokens from keycloak ?

Reply:
Understood, we are doing a soft migration.

Reply:
This func still used in some remote AuthZ tests.